modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -109,38 +150,106 @@ public void Dispose()
 
         // IList
 
+        /// <summary>
+        /// This will always be false. It's needed by <see cref="IList"/>.
+        /// </summary>
         public bool IsReadOnly => false;
 
+        /// <summary>
+        /// This will always be false. It's needed by <see cref="IList"/>.
+        /// </summary>
         public bool IsFixedSize => false;
 
+        /// <summary>
+        /// Returns the object at the given index.
+        /// </summary>
+        /// <value>The object at the given index.</value>
         public object this[int index]
         {
             get => godot_icall_Array_At(GetPtr(), index);
             set => godot_icall_Array_SetAt(GetPtr(), index, value);
         }
 
+        /// <summary>
+        /// Adds an object to the end of this <see cref="Array"/>.
+        /// This is the same as `append` or `push_back` in GDScript.
+        /// </summary>
+        /// <param name="value">The object to add.</param>
+        /// <returns>The new size after adding the object.</returns>
         public int Add(object value) => godot_icall_Array_Add(GetPtr(), value);
 
+        /// <summary>
+        /// Checks if this <see cref="Array"/> contains the given object.
+        /// </summary>
+        /// <param name="value">The item to look for.</param>
+        /// <returns>Whether or not this array contains the given object.</returns>
         public bool Contains(object value) => godot_icall_Array_Contains(GetPtr(), value);
 
+        /// <summary>
+        /// Erases all items from this <see cref="Array"/>.
+        /// </summary>
         public void Clear() => godot_icall_Array_Clear(GetPtr());
 
+        /// <summary>
+        /// Searches this <see cref="Array"/> for an object
+        /// and returns its index or -1 if not found.
+        /// Optionally, the initial search index can be passed.
```

Comment:
This method is missing the `int from` parameter that the GDScript `find` method has, so this description is incorrect.